content/en/blog/_posts/2020-12-02-dont-panic-kubernetes-and-docker.md

```diff
@@ -99,4 +99,6 @@ for you, that’s okay -- there’s a lot going on here, Kubernetes has a lot of
 moving parts, and nobody is an expert in 100% of it. We encourage any and all
 questions regardless of experience level or complexity! Our goal is to make sure
 everyone is educated as much as possible on the upcoming changes. `<3` We hope
-this has answered most of your questions and soothed some anxieties!
\ No newline at end of file
+this has answered most of your questions and soothed some anxieties!
+
+Looking for more answers? Check out our accompanying [Dockershim Deprecation FAQ](https://kubernetes.io/blog/2020/12/02/dockershim-faq/).
```

Comment:
```suggestion
Looking for more answers? Check out our accompanying [Dockershim Deprecation FAQ](/blog/2020/12/02/dockershim-faq/).
```

In the link, we need to use the path from the root because there is a case another domain is used.

Reply:
Fixed